xbmc/FileItem.cpp

```diff
@@ -3766,6 +3766,25 @@ bool CFileItem::LoadDetails()
     return false;
   }
 
+  if (IsVideo())
+  {
+    if (HasVideoInfoTag())
+      return true;
+
+    CVideoDatabase db;
+    if (!db.Open())
+    {
+      CLog::LogF(LOGERROR, "Error opening video database");
+      return false;
+    }
+
+    if (db.LoadVideoInfo(GetDynPath(), *GetVideoInfoTag()))
+      return true;
```

Comment:
you seem to return here and in line 3785 without closing the db connection too. Is this intentional?

Reply:
Also on the last line too

Reply:
See my other comment. This is intended because the close call will be done anyway in the dtor of CVideoDatabase.